lib/reline/ansi.rb

```diff
@@ -81,7 +81,7 @@ def self.set_screen_size(rows, columns)
 
   def self.cursor_pos
     begin
-      res = ''
+      res = String.new
```

Comment:
The unary plus is the shortest. Can we use that?

```suggestion
      res = +''
```


Here are some details:

```
      # In Ruby 2.3 or later, use unary plus operator to unfreeze a string
      # literal instead of `String#dup` and `String.new`.
      # Unary plus operator is faster than `String#dup`.
      #
      # Note: `String.new` (without operator) is not exactly the same as `+''`.
      # These differ in encoding. `String.new.encoding` is always `ASCII-8BIT`.
      # However, `(+'').encoding` is the same as script encoding(e.g. `UTF-8`).
      # So, if you expect `ASCII-8BIT` encoding, disable this cop.
```

https://github.com/rubocop-hq/rubocop-performance/blob/master/lib/rubocop/cop/performance/unfreeze_string.rb#L6-L13

Reply:
Ahh. I see. I wasn’t sure which ruby version to support and when `+””` arrived.

Reply:
https://github.com/ruby/reline/blob/master/reline.gemspec#L20

2.5+ Aha, cool. Supported!